src/scenekit.cs

```diff
@@ -160,6 +160,12 @@ interface SCNAnimatable {
 		[Export ("removeAllAnimations")]
 		void RemoveAllAnimations ();
 
+		[Introduced (PlatformName.MacCatalyst, 15, 0)]
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+		[Abstract]
```

Comment:
Adding an `[Abstract]` member to an existing protocol is a breaking change.

What we do is:

1. Add the `[Abstract]` inside a `#if XAMCORE_4_0` block
2. Add the xtro failure to the corresponding .ignore file, with a comment explaining why.